WPR-8978: Make 'scrollMode' default value as 'native' on Agate components (#748)
enactjs/agate: CHANGELOG.md

The following is a curated list of changes in the Enact agate module, newest changes on the top.

## [unreleased]

> Changed `scrollMode` default value as `native` on the following components: `Scroller`, `VirtualList` and `VirtualGridList`.

Reviewer: ```suggestion
- `agate/Scroller`, `agate/VirtualList` and `agate/VirtualGridList` default value for `scrollMode` to `native`.
```
